.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+> **Before You Begin**
+> If you have not already please read through the contributing guidelines
+
+## Checklist before requesting a review
+
+- [ ] I have followed the guidelines in the [Contributing document](../CONTRIBUTING.md)
+- [ ] I checked to make sure there aren't other open [Pull Requests](https://github.com/bitovi/use-simple-reducer/pulls) for the same update/change?
```

Comment:
I wonder if we could make a relative link work here? (so that there's nothing project-specific in the file)

Reply:
Oh interesting. I've never tried that on an md file. I'll try it real quick